data/Dockerfiles/watchdog/check_mysql_slavestatus.sh

```diff
@@ -132,9 +132,9 @@ fi
 
 # Connect to the DB server and store output in vars
 if [[ -n $socket ]]; then
-  ConnectionResult=$(mariadb --skip-ssl ${optfile} ${socket} ${user} -e "show slave ${connection} status\G" 2>&1)
+  ConnectionResult=$(mariadb --skip-ssl --host=${DBHOST} --port=${DBPORT} -u ${DBUSER} -p ${DBPASS} ${DBNAME} -e "show slave ${connection} status\G" 2>&1)
 else
-  ConnectionResult=$(mariadb --skip-ssl ${optfile} ${host} ${port} ${user} -e "show slave ${connection} status\G" 2>&1)
+  ConnectionResult=$(mariadb --skip-ssl --host=${DBHOST} --port=${DBPORT} -u ${DBUSER} -p ${DBPASS} ${DBNAME} -e "show slave ${connection} status\G" 2>&1)
 fi
```

Comment:
I think the variables used in the command (host, port, user) are set using the options names, see line 100 to 112. 

Searching the script name though the repo gives only 2 results :
- a copy in the watcher DockerFile
- a call in `watchdog.sh` 

As the script call in `watchdog.sh`  was already modified to have dbhost and dbport, I would recommend reverting this change here ? 


```suggestion
if [[ -n $socket ]]; then
  ConnectionResult=$(mariadb --skip-ssl ${optfile} ${socket} ${user} -e "show slave ${connection} status\G" 2>&1)
else
  ConnectionResult=$(mariadb --skip-ssl ${optfile} ${host} ${port} ${user} -e "show slave ${connection} status\G" 2>&1)
fi
```